src/orbit-common/cache.js

```diff
@@ -36,10 +36,11 @@ import 'orbit-common/rxjs/add/observable/from-orbit-event';
  @constructor
  */
 export default class Cache {
-  constructor(schema, options = {}) {
+  constructor(network, options = {}) {
```

Comment:
We should probably just accept `network` as an option and assert if it's not present, just as we do for sources. It's confusing that Cache has such a similar but slightly different constructor.